source/_docs/configuration/customizing-devices.markdown

```diff
@@ -71,9 +71,10 @@ homeassistant:
 
 ### {% linkable_title Device Class %}
 
-Device class is currently supported by the following platforms:
+Device class is currently supported by the following domains:
```

Comment:
Just wondering: I do understand the difference between platforms (`sensor.mqtt` for example) and components (`sensor`) - or at least I think I do.

But I've also heard the term "domain" a couple of times. Is a domain the same thing as component or rather the name of a component? Which one is preferred to be used?

Reply:
I think that we should avoid using domain here. In the user documentation we should stick to "component" and "platform".